pkg/ccl/importccl/exportcsv_test.go

```diff
@@ -263,6 +266,99 @@ func TestExportOrder(t *testing.T) {
 	}
 }
 
+// validateParquetFile reads the parquet file, converts each value in the parquet file to its
+// native go type, and asserts its values match the truth.
+
+// TODO (MB): once we figure out how to export the column names properly,
+// this function should check the column names as well
+func validateParquetFile(t *testing.T, file string, truthRows [][]interface{}) error {
+	r, err := os.Open(file)
+	if err != nil {
+		return err
+	}
+	defer r.Close()
+
+	fr, err := goparquet.NewFileReader(r)
+	if err != nil {
+		return err
+	}
+	t.Logf("Schema: %s", fr.GetSchemaDefinition())
+
+	cols := fr.SchemaReader.GetSchemaDefinition().RootColumn.Children
+	count := 0
+
+	for {
+		row, err := fr.NextRow()
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
+			return fmt.Errorf("reading record failed: %w", err)
+		}
+		if int64(count) > fr.NumRows() {
+			return fmt.Errorf("more row than accorrding to metadata")
+		}
+
+		t.Logf("\n Record %v:", count)
+		for i := 0; i < len(cols); i++ {
+			var decodedV interface{}
+			v := row[cols[i].SchemaElement.Name]
+			if vv, ok := v.([]byte); ok {
```

Comment:
nit: can we change to a switch statement, might be more readable.

switch t := v.(type) {
	case int64:
		// Use t to get deocdedV
	case string:
}